### PR TITLE
docs: constructors of TimeArray

### PR DESCRIPTION
**Pull request by iblis17**

### CODE REVIEW

File: docs/src/timearray.md [hunk 1]
```diff
@@ -70,3 +70,9 @@ values
 colnames
 meta
 ```
+
+## Constructors
```
Comment (femtotrader) [inline docs/src/timearray.md]: I wonder if doc for TimeArray constructors shouldn't be written before "Fields getter functions"

Maybe other functions which have a TimeArray as parameter should all be listed here (or in a separated page)

like http://juliadata.github.io/DataFrames.jl/latest/lib/functions.html

File: docs/Makefile [hunk 2]
```diff
@@ -1,2 +1,2 @@
 all:
-	julia --color=yes make.jl
+	julia --project=. --color=yes make.jl
```
Comment (femtotrader) [inline docs/Makefile]: I'd be very pleased to know why it's necessary (for my own information)

File: docs/src/timearray.md (hunk 1 above)
Comment (iblis17) [reply docs/src/timearray.md]: > I wonder if doc for TimeArray constructors shouldn't be written before "Fields getter functions"

okay, I can reagrange the sections.

>Maybe other functions which have a TimeArray as parameter should all be listed here (or in a separated page)

I think you want an API referece list, this is another issue.

File: docs/Makefile (hunk 2 above)
Comment (iblis17) [reply docs/Makefile]: Since we have `Project.toml` in that dir.

`julia --project=.` implies `julia -e 'using Pkg; Pkg.activate(".")'`

Comment (iblis17) [reply docs/Makefile]: And you can checkout the doc deployment instructions in `.travis.yml` as well

**Pull request merged**